_data/other.yml

```diff
@@ -49,6 +49,13 @@ websites:
       software: Yes
       doc: https://www.youtube.com/watch?v=5OLcvjVhhuI
 
+    - name: AwardWallet
+      url: https://www.awardwallet.com
```

Comment:
This URL redirects to one without a preceding `www`. Please update the url to `https://awardwallet.com`